dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -3474,6 +3486,146 @@ protected void checkLogsForErrorsAndWarnings(List<String> failuresRegExps, Strin
+                                                     "spanTestServer", //io.openliberty.microprofile.telemetry.internal.tests.JaegerBaseTest
+                                                     "spanTestServer", //io.openliberty.microprofile.telemetry.internal.tests.JvmMetricsOtelCollectorTest
+                                                     "spanTestServer", //io.openliberty.microprofile.telemetry.internal.tests.MetricsApiOtelCollectorTest
+                                                     "spanTestServer", //io.openliberty.microprofile.telemetry.internal.tests.TracingNotEnabledTest
+                                                     "spanTestServer", //io.openliberty.microprofile.telemetry.internal.tests.ZipkinOtelCollectorTest
+                                                     "spanTestServer", //io.openliberty.microprofile.telemetry.internal.tests.ZipkinTest
+                                                     "Health40TCKServer", //io.openliberty.microprofile.health40.tck.Health40TCKLauncher
+                                                     "ConfigAdminDropinsCheck", //io.openliberty.microprofile.health31.fat.ConfigAdminHealthCheckTest
+                                                     "ConfigAdminWrongAppCheck", //io.openliberty.microprofile.health31.fat.ConfigAdminHealthCheckTest
+                                                     "ConfigAdminXmlCheck", //io.openliberty.microprofile.health31.fat.ConfigAdminHealthCheckTest
+                                                     "DefaultStartupOverallStatusHealthCheck", //io.openliberty.microprofile.health31.fat.DefaultOverallStartupStatusUpAppStartupTest
+                                                     "FailedConfigAdminApplicationStateHealthCheck", //io.openliberty.microprofile.health31.fat.ConfigAdminHealthCheckTest
+                                                     "InvalidDefaultStartupOverallStatusProperty", //io.openliberty.microprofile.health31.fat.DefaultOverallStartupStatusUpAppStartupTest
+                                                     "SlowAppStartupHealthCheck", //io.openliberty.microprofile.health31.fat.SlowAppStartupHealthCheckTest
+                                                     "FATServer", //org.eclipse.microprofile.graphql.tck.GraphQLTckPackageTest
+    };
+    private static final Set<String> EXEMPT_SERVERS_SET = new HashSet<String>(Arrays.asList(EXEMPT_SERVERS));
+
+    /**
+     * After the server has shutdown, check that the features used at runtime matched those expected by any FeatureReplacementAction which may have been active.
+     *
+     * @throws Exception
+     */
+    protected void checkServerRepeatFeatures() throws Exception {
+        String method = "checkServerRepeatFeatures";
+
+        RepeatTestAction action = RepeatTestFilter.getMostRecentRepeatAction();
+        if (action != null && action instanceof FeatureReplacementAction) {
```

Comment:
`instanceof` always returns `false` for `null`, so the null check is not required.

Reply:
removed null check